src/atc/delta/autoloaderstream_handle.py

```diff
@@ -0,0 +1,61 @@
+from pyspark.sql import DataFrame
+
+from atc.configurator.configurator import Configurator
+from atc.spark import Spark
+from atc.tables import TableHandle
+from atc.tables.SparkHandle import DeltaHandleInvalidFormat
+
+
+class AutoloaderStreamHandle(TableHandle):
+    def __init__(
+        self,
+        *,
+        location: str,
+        checkpoint_path: str,
```

Comment:
@mrmasterplan we need to ensure that the checkpoint path is the same as the write checkpoint path: https://docs.databricks.com/getting-started/etl-quick-start.html#auto-loader